.github/workflows/ci.yml

```diff
@@ -22,6 +22,22 @@ jobs:
           command: test
           args: --verbose
 
+  build:
+    name: Build target ${{ matrix.target }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - wasm32-wasi
```

Comment:
Just curious – are we expecting to have additional cross-targets, or is this simply to have consistent variable declarations? (as opposed to using `env`)